src-tests/test-rpc.lisp

```diff
@@ -4,6 +4,20 @@
 
 (in-package #:rpcq-tests)
 
+;; TODO FIXME XXX The below relies on some funky/hacky/unsightly code
+;; to forcefully kill threads. See open issues
+;; https://github.com/rigetti/rpcq/issues/61
+;; https://github.com/rigetti/rpcq/issues/75
+
+(defun kill-thread-slowly (thread)
+  #+ccl
+  (loop :while (bt:thread-alive-p thread)
+        :do (sleep 1) (bt:destroy-thread thread))
+  #-ccl
+  (when (bt:thread-alive-p thread)
+    (bt:destroy-thread thread))
+  (sleep 1))
```

Comment:
I'm curious about why do you need to wait, then kill in CCL whereas you kill, then wait on other compilers.

Reply:
The CCL sleep was pre-existing. I considered flipping the order, but decided to let a sleeping dog lie 🥁 .

Possibly the single sleep would be enough to also fix CCL shutdown, but I didn't test it.